pkg/rpc/client/policy.go

```diff
@@ -43,9 +34,6 @@ func (c *Client) GetMaxNotValidBeforeDelta() (int64, error) {
 
 // invokeNativePolicy method invokes Get* method on a native Policy contract.
 func (c *Client) invokeNativePolicyMethod(operation string) (int64, error) {
-	if !c.cache.initDone {
-		return 0, errNetworkNotInitialized
```

Comment:
Why are you removing the check here?

Reply:
Because we don't really need client initialisation for this. We have a nice `c.GetNativeContractHash` method which manages cache and if the Policy hash is missing then we can ask the node for it. It is fixed in a separate commit.

Reply:
Yeah, some of these removal depend on subsequent commits which is not exactly the best way to structure them. It'd be nice to have something consistent after every commit.

Reply:
Fixed, I've grouped cache-related commits in a single commit, so now the code is consistent after every commit.